### ISSUE TITLE
healthcheck does not recognize ping configuration

### ISSUE THREAD

**Issue by jansauer** [containous/traefik#5226]
## Do you want to request a *feature* or report a *bug*?

Bug

### Did you try using a 1.7.x configuration for the version 2.0?

- [ ] Yes
- [X] No

### What did you do?

I tried adding a health check to my traefik docker setup. After learning that curl is not included in the alpine image my goal was to use the `healthcheck` cli command. Unfortunately the command always fails with the error message the i'm allegedly don't have the ping enabled.
My configuration can be reproduce/dumped down using...  

```
$ docker run -d \
  --name traefik \
  --publish 80:80 \
  --publish 8080:8080 \
  --restart unless-stopped \
  traefik:v1.7.14-alpine \
    --ping \
    --api \
    --loglevel=DEBUG
$ docker exec -it traefik traefik healthcheck
Error calling healthcheck: please enable `ping` to use health check
```

Based on the debug log output and the fact that i can call `/ping` via curl i'm sure that ping is enabled.

```
$ curl http://localhost:8080/ping
OK
```

### What did you expect to see?

A 0 exit code.

### What did you see instead?

`Error calling healthcheck: please enable `ping` to use health check` with exit code 1.

### Output of `traefik version`: (_What version of Traefik are you using?_)

```
traefik:v1.7.14-alpine
Version:      v1.7.14
Codename:     maroilles
Go version:   go1.12.8
Built:        2019-08-14_09:46:58AM
OS/Arch:      linux/amd64
```

### What is your environment & configuration (arguments, toml, provider, platform, ...)?

All configuration is provided via `docker run` command arguments.

### If applicable, please paste the log output in DEBUG level (`--log.level=DEBUG` switch)

```
$ docker logs traefik
time="2019-08-19T08:47:23Z" level=info msg="Traefik version v1.7.14 built on 2019-08-14_09:46:58AM"
time="2019-08-19T08:47:23Z" level=debug msg="Global configuration loaded {\"LifeCycle\":{\"RequestAcceptGraceTimeout\":0,\"GraceTimeOut\":10000000000},\"GraceTimeOut\":0,\"Debug\":false,\"CheckNewVersion\":true,\"SendAnonymousUsage\":false,\"AccessLogsFile\":\"\",\"AccessLog\":null,\"TraefikLogsFile\":\"\",\"TraefikLog\":null,\"Tracing\":null,\"LogLevel\":\"DEBUG\",\"EntryPoints\":{\"http\":{\"Address\":\":80\",\"TLS\":null,\"Redirect\":null,\"Auth\":null,\"WhitelistSourceRange\":null,\"WhiteList\":null,\"Compress\":false,\"ProxyProtocol\":null,\"ForwardedHeaders\":{\"Insecure\":true,\"TrustedIPs\":null}},\"traefik\":{\"Address\":\":8080\",\"TLS\":null,\"Redirect\":null,\"Auth\":null,\"WhitelistSourceRange\":null,\"WhiteList\":null,\"Compress\":false,\"ProxyProtocol\":null,\"ForwardedHeaders\":{\"Insecure\":true,\"TrustedIPs\":null}}},\"Cluster\":null,\"Constraints\":[],\"ACME\":null,\"DefaultEntryPoints\":[\"http\"],\"ProvidersThrottleDuration\":2000000000,\"MaxIdleConnsPerHost\":200,\"IdleTimeout\":0,\"InsecureSkipVerify\":false,\"RootCAs\":null,\"Retry\":null,\"HealthCheck\":{\"Interval\":30000000000},\"RespondingTimeouts\":null,\"ForwardingTimeouts\":null,\"AllowMinWeightZero\":false,\"KeepTrailingSlash\":false,\"Web\":null,\"Docker\":null,\"File\":null,\"Marathon\":null,\"Consul\":null,\"ConsulCatalog\":null,\"Etcd\":null,\"Zookeeper\":null,\"Boltdb\":null,\"Kubernetes\":null,\"Mesos\":null,\"Eureka\":null,\"ECS\":null,\"Rancher\":null,\"DynamoDB\":null,\"ServiceFabric\":null,\"Rest\":null,\"API\":{\"EntryPoint\":\"traefik\",\"Dashboard\":true,\"Debug\":false,\"CurrentConfigurations\":null,\"Statistics\":null},\"Metrics\":null,\"Ping\":{\"EntryPoint\":\"traefik\"},\"HostResolver\":null}"
time="2019-08-19T08:47:23Z" level=info msg="\nStats collection is disabled.\nHelp us improve Traefik by turning this feature on :)\nMore details on: https://docs.traefik.io/basics/#collected-data\n"
time="2019-08-19T08:47:23Z" level=info msg="Preparing server http &{Address::80 TLS:<nil> Redirect:<nil> Auth:<nil> WhitelistSourceRange:[] WhiteList:<nil> Compress:false ProxyProtocol:<nil> ForwardedHeaders:0xc00033c260} with readTimeout=0s writeTimeout=0s idleTimeout=3m0s"
time="2019-08-19T08:47:23Z" level=info msg="Preparing server traefik &{Address::8080 TLS:<nil> Redirect:<nil> Auth:<nil> WhitelistSourceRange:[] WhiteList:<nil> Compress:false ProxyProtocol:<nil> ForwardedHeaders:0xc00033c280} with readTimeout=0s writeTimeout=0s idleTimeout=3m0s"
time="2019-08-19T08:47:23Z" level=info msg="Starting provider configuration.ProviderAggregator {}"
time="2019-08-19T08:47:23Z" level=info msg="Starting server on :80"
time="2019-08-19T08:47:23Z" level=info msg="Starting server on :8080"
```

**Comment by ldez** [comment]
Hello,

you have to enable the ping on the healthcheck command.

```
docker exec -it traefik traefik healthcheck --api
```

**Comment by jansauer** [comment]
@ldez thx for the tip. Haven't thought of that since the server itself has `--ping` enabled. But since the one-off cli command does not share any configuration with the server... 🤔

```
$ docker exec -it traefik traefik healthcheck --ping
OK: http://:8080/ping
```

Works 🍾🥳

Nevertheless the cli does not behave as documented. Based on the help message the command usage is as follows `traefik [flags] <command> [<arguments>]`. `--ping` is listed as flag but putting it in front of `healthcheck` command fails.

```
$ docker exec -it traefik traefik --ping healthcheck
FATA[2019-08-20T06:34:42Z] Error preparing server: error opening listener: listen tcp :80: bind: address already in use
```

Also the command description in the help message should be changed from
`Calls traefik /ping to check health (web provider must be enabled)` 
to
`Calls traefik /ping to check health (ping must be enabled)`.

**Comment by SantoDE** [comment]
Hey @jansauer,

good catch! Would you be willing to update the docs in a pr? :)

**Comment by jansauer** [comment]
Apparently this was already done on master. The current text is

```
Usage: traefik [command] [flags] [arguments]
```

@SantoDE i'm not sure if and who to backport changes for 1.7 🤔

**Comment by SantoDE** [comment]
There is a 1.7 branch @jansauer, just let your PR point to that :)

**Comment by traefiker** [comment]
Closed by #5233.
